flow/project.py

```diff
@@ -2397,7 +2374,9 @@ def _dumps_op(op):
     def _loads_op(self, blob):
         id, name, job_ids, cmd, directives = blob
         jobs = tuple(self.open_job(id=job_id) for job_id in job_ids)
-        return _JobOperation(id, name, jobs, cmd, directives)
+        base_directives = jobs[0]._project._environment._get_default_directives()
+        base_directives.update(directives)
```

Comment:
Same as before - is `base_directives` the right name for something that is updated?